geps/gep-3779/index.md

```diff
@@ -0,0 +1,263 @@
+# GEP-3779: Identity Based Authz for East-West Traffic
+
+* Issue: [#3779](https://github.com/kubernetes-sigs/gateway-api/issues/3779)
+* Status: Provisional
+
+(See [status definitions](../overview.md#gep-states).)
+
+
+## TLDR
+
+Provide a method for configuring Gateway API Mesh implementations to enforce east-west identity-based Authorization controls. At the time of writing this we leave Authentication for specific implementation and outside of this proposal scope.
+
+
+## Goals
```

Comment:
Is any attributes other than identity and port considered?

Reply:
this would mean entering the L7 space (which I think require more research about where we can standardize). So for the first iteration -- no.

Reply:
I suggest making this clear in the non-goals, and then we can consider this resolved.